cmd/util.go

```diff
@@ -336,9 +336,14 @@ func runAudit(auditFunc interface{}) func(cmd *cobra.Command, args []string) {
 			log.Error(err)
 			return
 		}
+		occurrenceCount := 0
 		results := getResults(resources, auditFunc)
 		for _, result := range results {
 			result.Print()
+			occurrenceCount += len(result.Occurrences)
```

Comment:
I don't think we want to return 1 in the case where kubeaudit returns a bunch of Occurrences with level `warn` or `info`, maybe you should strictly check for level `error` in this case, using a bool flag in place of occurrenceCount is much better.